.github/workflows/kubeapps-main-reusable.yaml

```diff
@@ -0,0 +1,754 @@
+          fi
+
+          if [[ ${GITHUB_REF_TYPE} == "tag" && ${GITHUB_REF_NAME} =~ ^v[0-9]+ ]]; then
+            echo "running_on_tag=true" >> $GITHUB_OUTPUT
+          else
+            echo "running_on_tag=false" >> $GITHUB_OUTPUT
+          fi
+
+          echo "dev_mode=${DEV_MODE}" >> $GITHUB_OUTPUT
+          echo "ssh_key_kubeapps_deploy_filename=${SSH_KEY_KUBEAPPS_DEPLOY_FILENAME}" >> $GITHUB_OUTPUT
+          echo "ssh_key_forked_charts_deploy_filename=${SSH_KEY_FORKED_CHARTS_DEPLOY_FILENAME}" >> $GITHUB_OUTPUT
+          echo "img_modifier=${IMG_MODIFIER}" >> $GITHUB_OUTPUT
+          echo "img_dev_tag=${IMG_DEV_TAG}" >> $GITHUB_OUTPUT
+          echo "postgresql_version=${POSTGRESQL_VERSION}" >> $GITHUB_OUTPUT
+          echo "rust_version=${RUST_VERSION}" >> $GITHUB_OUTPUT
+      - name: Show outputs
+        run: |
+          echo "DEV_MODE: ${{steps.set-outputs.outputs.dev_mode}}"
+          echo "IMG_MODIFIER: ${{steps.set-outputs.outputs.img_modifier}}"
+          echo "IMG_PREFIX: ${{steps.set-outputs.outputs.img_prefix}}"
+          echo "IMG_DEV_TAG: ${{steps.set-outputs.outputs.img_dev_tag}}"
+          echo "IMG_PROD_TAG: ${{steps.set-outputs.outputs.img_prod_tag}}"
+          echo "POSTGRESQL_VERSION: ${{steps.set-outputs.outputs.postgresql_version}}"
+          echo "RUST_VERSION: ${{steps.set-outputs.outputs.rust_version}}"
+          echo "RUNNING_ON_MAIN: ${{steps.set-outputs.outputs.running_on_main}}"
+          echo "RUNNING_ON_TAG: ${{steps.set-outputs.outputs.running_on_tag}}"
+          echo "SSH_KEY_KUBEAPPS_DEPLOY_FILENAME: ${{steps.set-outputs.outputs.ssh_key_kubeapps_deploy_filename}}"
+          echo "SSH_KEY_FORKED_CHARTS_DEPLOY_FILENAME: ${{steps.set-outputs.outputs.ssh_key_forked_charts_deploy_filename}}"
+          echo "TRIGGERED_FROM_FORK: ${{steps.set-outputs.outputs.triggered_from_fork}}"
+          echo "VERSION: ${{steps.set-outputs.outputs.version}}"
+
+  test_go:
+    needs:
+      - setup
+    runs-on: ubuntu-latest
+    services:
+      postgresql:
+        image: bitnami/postgresql:${{needs.setup.outputs.postgresql_version}}
+        options: >-
+          --health-cmd "pg_isready -U postgres"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+        env:
+          ALLOW_EMPTY_PASSWORD: "yes"
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: ${{ env.GOLANG_VERSION }}
+    - name: Run go unit tests
+      run: make test
+    - run: make test-db
+
+  test_dashboard:
+    runs-on: ubuntu-latest
+    needs:
+      - setup
+    env:
+      # Note that the max old space setting is per worker, so running the tests
+      # with 4 workers on a 4Gb (free plan) needs 1Gb of max old space. Forcing
+      # garbage collection to start earlier with 512M per worker.
+      NODE_OPTIONS: "--max-old-space-size=512"
```

Comment:
Is this assumption still true for GHA runners?

Reply:
To be honest, I don't know 😅 I copy&pasted it from the CircleCI setup and it works, so I haven't tried to change it. If you consider it is needed, we can add an issue to fine-tune its value after the migration.